index.html

```diff
@@ -309,6 +309,76 @@ <h2 class="heading-primary">About Myself</h2>
         </div>
       </div>
     </section>
+
+    <section class="contact" id="contact">
+      <img
+        src="images/section-about/contact-form-shapes-mobile.svg"
+        alt="background"
+        class="contact-bg"
+      />
+      <div class="contact-text">
+        <h3 class="heading-tertiary">Contact me</h3>
+        <p class="secondary-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form
```

Comment:
Please add image decorations spotted in the screenshot in the right place of the form
![images](https://user-images.githubusercontent.com/35504149/134144606-3a016b86-e27f-4992-89fe-924ff71000a7.png)
.

Reply:
Done